contents/tutorials/power-users.md

```diff
@@ -0,0 +1,72 @@
+---
+title: How to identify and analyze power users
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-10-26
+featuredImage: ../images/tutorials/banners/power-users.png
+topics: ['trends', 'cohorts']
+---
+
+Power users are users who use your product the most, significantly more than the average user. They have different needs and usage patterns than average users. Often, they are a company’s best and most important customers.
+
+Power users are knowledgeable, willing to learn, and likely to give feedback about your product. They care more about new features and upgrades than the average user does. Changes can impact a power user while not impacting an average user. That is why it is important to analyze power users and build for them as a group separately.
+
+Building a better product for power users keeps them happy and helps prevent them from churning. It helps you build a competitive advantage by growing lock-in with these users. It also  encourages average users to increase their usage and become power users.
+
+To create a great product for power users, you must know who they are and how they are using your product. In this tutorial, we’ll cover how to identify and analyze power users. All you need to start is a PostHog instance (and some users).
+
+## Defining a power user
+
+The first step to identifying and analyzing a power user is defining them. There is no one-size-fits-all, every company and product has its own definition. Defining what they do is key to identifying and analyzing them. Ideally, your power users should align with your ideal customer profile and your key product metrics.
+
+One common aspect between definitions of power users is high usage, whether that is visiting every day, using feature lots, or long session times. A power user relies on the product, it is core to their workflow. They also use advanced features more than average.
+
+To help define exactly what a power user looks like, you can use PostHog. To do so, identify the key features for power users and analyze the events and actions related to those features. Some ideas:
+
+- Compare the retention percentage or returning amounts. Power users have higher retention.
+
+- Compare the stickiness (what events are causing users to come back repeatedly). Power users use features repeatedly.
+
+- Create trends for average user usage vs 90th, 95th, and 99th percentile usage. Power users use features significantly more than average. The larger the gap, the better.
+
+If a feature shows higher retention, repeated use, and a large difference in usage to the average, it is likely valuable to a power user. You can use data related to this feature along with qualitative analysis to solidify the definition of your power user.
```

Comment:
I feel like we need an example here. Can we use an example from Posthog – i.e. PostHog power users to X more often than non-powers, and this is how we worked that out.

Reply:
Added a little bit more on user interviews and session recordings